src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/8.diff.sql

```diff
@@ -0,0 +1,411 @@
+/*************************************************************
+    Migration 8: Adds Version and Fragment columns
+    to UriSearchParam to support Canonical searches
+**************************************************************/
+
+BEGIN TRANSACTION
+
+CREATE TYPE dbo.UriSearchParamTableType_2 AS TABLE
+(
+    SearchParamId smallint NOT NULL,
+    Uri varchar(256) COLLATE Latin1_General_100_CS_AS NOT NULL,
+    Version varchar(64) COLLATE Latin1_General_100_CS_AS NULL,
+    Fragment varchar(256) COLLATE Latin1_General_100_CS_AS NULL
+)
+
+GO
+
+ALTER TABLE dbo.UriSearchParam
+    ADD Version varchar(64) COLLATE Latin1_General_100_CS_AS NULL,
+        Fragment varchar(256) COLLATE Latin1_General_100_CS_AS NULL;
+
+GO
+
+CREATE NONCLUSTERED INDEX IX_UriSearchParam_SearchParamId_UriVersionFragment
```

Comment:
You need to drop the old index

Reply:
Another option is to keep the same index name add 
`WITH (DATA_COMPRESSION = PAGE, DROP_EXISTING = ON, ONLINE = ON)`